lab-michael/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const HTMLPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: `${__dirname}/app/index.js`,
+  output: {
+    filename: 'bundle.js',
+    path: 'build',
```

Comment:
I believe this is a broken config with webpack 2.x. The value of path needs to be an absolute path.